html/user/get_project_config.php

```diff
@@ -138,6 +138,13 @@ function show_platforms() {
     readfile("../../project_keywords.xml");
 }
 
+$keydir = parse_config($config, "<key_dir>");
+if (is_readable($keydir."/account_ownership_public.pem")) {
+    echo "    <account_ownership_public_key>";
+    echo base64_encode(file_get_contents($keydir."/account_ownership_public.pem"));
```

Comment:
the references to account_ownership_public.pem should be based on a variable created by the include file that contains the other variables for referencing these files (maybe create inc/account_ownership.inc?).  The file names should be defined exactly once in the code base.